docs/advanced-features/instrumentation.md

```diff
@@ -0,0 +1,37 @@
+---
+description: Learn how instrument your Next.js app.
+---
+
+> **Note**: This feature is experimental, you need to explicitly opt-in by providing `experimental.instrumentationHook = true;` in your `next.config.js`.
+
+# `instrumentation.ts`
+
+You can export `register` function from this file and we will be called whenever we bootstrap a new server instance.
+When deployed it will be called continuously on each coldboot (but exactly once in each environment).
+
+Many people used imports with side-effects before we released this feature.
+The behavior is similar, but using imports with side-effects is hard to work with and has many down-sides.
```

Comment:
```suggestion
The behavior is similar, but using imports with side-effects is hard to work with and has many downsides.
```

What do you mean by "The behavior is similar" here? Are you referring to the behavior of `register`?

Reply:
Yes, I want to say that `register` does the same thing. But better and more robust/reliable.